pkg/provisioners/message_dispatcher.go

```diff
@@ -20,12 +20,12 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/knative/eventing/pkg/utils"
```

Comment:
Prefer to put these external imports in a separate (lower) block, rather than sorting them with the go builtin libraries.

Reply:
done